ci/pipelines/perf-tests.yml

```diff
@@ -87,6 +87,24 @@ resources:
       location: Europe/London
     type: time
 
+  - name: every-sunday-at-0315
```

Comment:
I think it would be better if the name told you what it does rather than when it does it.

Reply:
What it does though is dependent on what it's connected to. You can connect as many things as you want to this. If you see it in the context of the pipeline 
![Screenshot 2023-11-02 at 16 41 29](https://github.com/alphagov/pay-ci/assets/2170030/ac1aa43d-7b24-4558-a27e-df94c230cc82)
It's a lot more useful to have it say when, otherwise you can't tell it's a timer.

We have this same nomenclature in various places:

![Screenshot 2023-11-02 at 16 42 10](https://github.com/alphagov/pay-ci/assets/2170030/f497ea9d-3712-4c1f-8de6-1ecfb6f26a75)
![Screenshot 2023-11-02 at 16 42 23](https://github.com/alphagov/pay-ci/assets/2170030/219090c8-369b-4313-b4af-2f82b23a1bfe)